essentials.yml

```diff
@@ -0,0 +1,4 @@
+---
+ath:
+  useLocalSnapshots: false
+  athRevision: "b36f2d27fdaa3f7ce73f0179ddbd26a6d7fa05fc"
```

Comment:
Probably you want https://github.com/jenkinsci/acceptance-test-harness/commit/06caf67d37d945ec5228a80d73714fac692c28d3 now?

Reply:
Yeah, that was merged after me creating this PR, I was watching it in any case and that was the reason my PR to be downstream of the source of that commit, I am going to update it with the rest of the changes requested in the review of #3347 .

In any case, I have not yet decided to use this commit id in the final version (which exact version is still open to discussion as there is no one yet taking the responsibility to update it), will just use it until my code meets the requirements and then update it to the proper version